### DISCUSSION TITLE
how do i add the trader profession? love your work<3

### DISCUSSION THREAD

**Discussion by thecounter89** [samolego/Taterzens#76] [Q&A] [answered]
i think all would benefit from a lil tutorial

**Comment by thecounter89** [comment, marked as answer]
i figured it out anyone else who is confused this is the mod download for trading npcs:
https://github.com/samolego/TraderNPCs/releases/download/1.1.0/tradernpcs-1.1.0-fabric.jar

**Comment by samolego** [comment]
Make sure to read https://samolego.github.io/Taterzens/1.8.0/professions/traders/ :)
